src/pages/networks/NetworkDetail.tsx

```diff
@@ -47,15 +50,15 @@ const NetworkDetail: FC = () => {
   }
 
   const isManagedNetwork = network?.managed;
+  const isFeatureSupported =
+    !typesWithForwards.includes(network?.type ?? "") || !isManagedNetwork;
+  const isPeeringSupported = network?.type === ovnType;
 
-  const getTabs = () => {
-    const type = network?.type ?? "";
-    if (!typesWithForwards.includes(type) || !isManagedNetwork) {
-      return ["Configuration"];
-    }
+  const tabLink = `/ui/project/${encodeURIComponent(project)}/network/${encodeURIComponent(name)}`;
 
-    return ["Configuration", "Forwards", "Leases"];
-  };
+  const tabClassnames = classnames("p-tabs__link", {
+    "is-disabled": isFeatureSupported,
+  });
```

Comment:
```suggestion
  const getTabClassnames = (isDisabled: boolean) => classnames("p-tabs__link", {
    "is-disabled": isDisabled,
  });
```
Let's make this a callback, so we can call it as `getTabClassnames(!hasForwards)`, or `getTabClassnames(!hasLeases)` and `getTabClassnames(!isPeeringSupported)`